platalea/dataset.py

```diff
@@ -55,8 +56,12 @@ def __init__(self, root, feature_fname, split='train', language='en',
             self.text_key = 'raw_jp'
         else:
             raise ValueError('Language {} not supported.'.format(language))
+        self.root = root
+        self.split = split
+        self.language = language
         root_path = pathlib.Path(root)
-        # Loading metadata
+        with open(root_path / 'label_encoders.pkl', 'rb') as f:
```

Comment:
Where is this pickle file saved?

Reply:
you can find a version under /home/bjrhigy/corpora/flickr8k/label_encoders.pkl or in flickr1d repo.